README.md

```diff
@@ -106,143 +106,7 @@ postgres=# \dt
-
-`POST`
-
-- Data Params
-
-```
-{
-    "code": "githubcode"
-}
-```
-
-- Success Response:
-    - Code: 200
-    - Content: `{"login":"","avatar_url":"","name":"","token":""}`
-
-- Error Response:
-
-    - Code: 404 NOT FOUND
-
-        Content: `{ error : "User doesn't exist" }`
-    
-    OR
-
-    - Code: 401 UNAUTHORIZED
-
-        Content: `{ error : "You are unauthorized to make this request." }`
-### Simple Auth
-
-- URL
-
-/altauth
-
-- Method:
-
-`POST`
-
-#### Api Key Header
-`x-api-key: <API_KEY>`
-
-- Success Response:
-    - Code: 200
-    - Content: `{Valid Api Key}`
-
-- Error Response:
-
-    - Code: 401 UNAUTHORIZED
-
-        Content: `{ error : "error validating api key" }`
-### Create Service
-`/createservice` - Creates a new database service. Only Postgres databases are supported at the moment
-#### Authentication Header
-`Bearer <GITHUB_TOKEN>`
-### Api Key Header
-`x-api-key: <API_KEY>`
-#### Method:
-
-`POST`
-
-#### Data Params
-- `userId=[String]`: GitHub username of the user.(Necessary only if using github authentication)
-- `version=[Object]`: Preferred database version. It must have the following properties:
-    - `maj=[Integer]`: Database major version. E.g for Postgres:9.6, the value here is 9.
-    - `min=[Integer]`: Database major version. E.g for Postgres:9.6, the value here is 6.
-- `db=[Object]`: A JSON object describing the properties of the database to be created. The properties are:
-  - `type=[String]`: Type of the database service to be created. Only `postgres` is supported for now.
-  - `name=[String]`: Name of the database service
-  - `monitoring=[String]`: Optional. Must be either of `enable` or `disable` when present. When enabled, it turns on prometheus metrics for the database
-    on port 9187.
-```json
-{
-  "userId": "viggy28",
-  "db": {
-    "type": "postgres",
-    "name": "localtest",
-    "monitoring": "enable"
-  },
-  "version": {
-    "maj": 9,
-    "min": 6
-  }
-}
-```
-
-#### Success Response:
-    Code: 200
-
-#### Error Response:
-    Code: 500 INTERNALSERVER ERROR
-
-### Get service
-`/cluster?cluster_id=<CLUSTER_ID>` - Fetches specific details of a service or cluster
-#### Method:
-`GET`
-#### Authentication Header
-`Bearer <GITHUB_TOKEN>`
-#### Api Key Header
-`x-api-key: <API_KEY>`
-#### Path Parameters
-- `cluster_id=[String]`: ID of the cluster to be retrieved.
-#### Responses
-200:
-```json
-{
-    "data": {
-        "cluster_id": "a130e42a09f4\n",
-        "id": 2,
-        "name": "localtest",
-        "port": 5432,
-        "username": "localtest"
-    }
-}
-```
-404:
-```json
-{
-    "message": "no cluster found with matching id"
-}
-```
-
-### JWT
-- URL
-
-/jwt?data=replaceme
-
-- Method:
-
-`GET`
-
-- Success Response:
-    - Code: 200
-    - Content: `{jwtofreplaceme}`
+The API documentation is autogenerated and saved in `openapi/swagger.yaml`. To view the documentaion on Swagger UI, run the command below from the project directory:
+```bash
+make serve-docs
```

Comment:
serve-docs expects the openapi/swagger.yaml. Should we call `docs` target first when `serve-docs` is invoked?

Also, I had to install `swagger` which I did through `make install-deps`. (Not sure whether we need to mention this)

Reply:
not really, but we can mention that. The docs is added to git so `make serve-docs` will work. `make docs` is needed when the documentation needs to be regenerated.

Reply:
makes sense.